### PR TITLE
Windows raise widget example

### PR DESCRIPTION
**Pull request by samsonwang**
Bring widget to the front when second instance is started, this is a workaround for Windows system.

### CODE REVIEW

**Comment by samsonwang** [comment]
The second instance should call `AllowSetForegroundWindow` before the first instance calls `SetForegoundWindow`.
The `instanceStared` signal is emitted once the second instance is created.
So, the second instance have to use `sendMessage` to notify the first instance.

File: examples/windows_raise_widget/main.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    QObject::connect(&app, &SingleApplication::instanceStarted,
+                     widget, [widget] () { RaiseWidget(widget); } );
+#endif
+
+    widget->show();
+
+    return app.exec();
+}
+
+void RaiseWidget(QWidget* widget) {
```
Comment (Shatur) [inline examples/windows_raise_widget/main.cpp:44]: We use camelCase for function names in this project. Could you rename it into `raiseWidget`?

File: Windows.md [hunk 2]
```diff
@@ -7,36 +7,46 @@ application can bring it's primary instance window whenever a second copy
...
-include `Mode::SecondaryNotification`, See `SingleApplication::Mode` for more
-details.
+`allowSecondary` parameter set to `true` .
 
-Here is an example:
+If the widget is minimized to Windows task bar, `QWidget::raise()` or
+`QWidget::show()` can not bring it to the front, you have to use Windows API
+`ShowWindow()` .
+
+Here is an example, you can find this code in the example directory:
```
Comment (Shatur) [inline Windows.md:21]: Instead of copying code I would just add link to the example.

**Review by Shatur — CHANGES_REQUESTED** [review]
Thanks!

File: Windows.md [hunk 3]
```diff
@@ -7,36 +7,15 @@ application can bring it's primary instance window whenever a second copy
...
-    );
-}
-```
-
-```cpp
-void App::instanceStarted() {
-    QApplication::setActiveWindow( [window/widget to set to the foreground] );
-}
-```
+You can find demo code in the [examples](examples/windows_raise_widget/main.cpp) directory.
```
Comment (Shatur) [inline Windows.md:21]: One minor nitpick and we are good to go:
```suggestion
You can find [demo code](examples/windows_raise_widget/main.cpp) in the examples directory.
```

Comment (samsonwang) [reply Windows.md:21]: It looks better.

**Comment by itay-grudev** [comment]
Looks good to me. @Shatur I think you need to approve the changes you requested.

**Review by Shatur — APPROVED** [review]

**Pull request merged**